<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab7CamOpencv</class>
 <widget class="QWidget" name="Tab7CamOpencv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="plabelCamView">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Images.qrc">:/Images/Images/display.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2">
       <item>
        <widget class="QPushButton" name="pPBcamStart">
         <property name="text">
          <string>CamStart</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pCBrgb">
         <property name="text">
          <string>RGB Classify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pPBsnapShot">
         <property name="text">
          <string>Snapshot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
